vpr/src/base/netlist_writer.cpp

```diff
@@ -2124,6 +2326,23 @@ void netlist_writer(const std::string basename, std::shared_ptr<const AnalysisDe
     nl_walker.walk();
 }
 
+///@brief Main routing for this file. See netlist_writer.h for details.
+void merged_netlist_writer(const std::string basename, std::shared_ptr<const AnalysisDelayCalculator> delay_calc, struct t_analysis_opts opts) {
```

Comment:
I think "Main routing" should be "Main routine"

Reply:
Fixed typo